auto_nag/scripts/workflow/no_priority.py

```diff
@@ -32,6 +32,15 @@ def description(self):
     def nag_template(self):
         return self.template()
 
+    def nag_preamble(self):
+        return """<p>
+  <b>You're getting this email because some bugs where you're the triage owner or the triage owner deputy have no priority set.</b>
```

Comment:
```suggestion
  <b>You're getting this email because some bugs for which you're the triage owner or the triage owner deputy have no priority set.</b>
```